src/main/java/bio/terra/app/controller/SnapshotsApiController.java

```diff
@@ -159,12 +159,13 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
 
   @Override
   public ResponseEntity<JobModel> exportSnapshot(
-      @PathVariable("id") UUID id, Boolean exportGsPaths) {
+      @PathVariable("id") UUID id, Boolean exportGsPaths, Boolean validatePrimaryKeyUniqueness) {
```

Comment:
I'm curious: do these parameters need to be not-primitives?  Like what happens if those are null and then are used to call `snapshotService.exportSnapshot`?  I think we might get an NPE no?  Might be worth double checking that they get properly set to false when no value is passed in (and if that's the case, I'd make them primitives to avoid confusion)

Reply:
The generated interface method uses boxed types, so these need to be boxed as well. I think we're assuming that they won't be null, and that Spring will enforce that.

Reply:
Ahhhh gotcha.  Yep, that totally makes sense